lib/Sema/TypeCheckAttr.cpp

```diff
@@ -3609,12 +3610,14 @@ static IndexSubset *computeDifferentiabilityParameters(
 // If the function declaration cannot be resolved, emits a diagnostic and
 // returns nullptr.
 static AbstractFunctionDecl *findAbstractFunctionDecl(
```

Comment:
`findAbstractFunctionDecl` is in need of a revamp, diagnostics are currently handled messily. I'll look into it after this PR lands.